king_phisher/server/database/models.py

```diff
@@ -397,6 +398,105 @@ class Message(Base):
 	credentials = sqlalchemy.orm.relationship('Credential', backref='message', cascade='all, delete-orphan')
 	visits = sqlalchemy.orm.relationship('Visit', backref='message', cascade='all, delete-orphan')
 
+@register_table
+class TestQuestion(Base):
+	__repr_attributes__ = ('question',)
+	__tablename__ = 'test_questions'
+	id = sqlalchemy.Column(sqlalchemy.Integer, primary_key=True)
+	question = sqlalchemy.Column(sqlalchemy.String, nullable=False)
+	question_type = sqlalchemy.Column(sqlalchemy.String, nullable=False)
+	description = sqlalchemy.Column(sqlalchemy.String)
+	hint = sqlalchemy.Column(sqlalchemy.String)
+	url_reference = sqlalchemy.Column(sqlalchemy.String)
+	url_hint = sqlalchemy.Column(sqlalchemy.String)
+	# relationships
+	submission_answer_link = sqlalchemy.orm.relationship('TestSubmissionLinkTestAnswer', backref='test_questions', cascade='all, delete-orphan')
```

Comment:
When you change this to a SQLAlchemy association, this can be renamed to simply `submissions` since we'd be removing the intermediary object which would also make it cleaner from a Python perspective.